var/www/html/tools/MemcachedAdmin/Library/Analysis.php

```diff
@@ -0,0 +1,332 @@
+<?php
+/**
+ * Copyright 2010 Cyrille Mahieux
```

Comment:
well that is a good question, I had initially intended these memcached and opcache debugging tools to be temporary, I just added them so I could ensure memcached and opcache were working as expected.

Should I go ahead and pull them out and clean up this templating? Or would you like to keep them?